packages/quantic/force-app/main/default/lwc/quanticTab/quanticTab.css

```diff
@@ -0,0 +1,5 @@
+.tab_button{
```

Comment:
Did this fix the tab bar collapse problem? Showing 1 less tab than what was expected?

Reply:
Yes, the tabs now do collapse accordingly when width is decreased.

Reply:
Yes, tabs now collapse accordingly when width is decreased